package.json

```diff
@@ -18,7 +19,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "prettier --write \"src/**/*.{js,jsx,css,html,scss}\""
```

Comment:
We can add additional scripts to check the format. e.g.:
`"format:check": "prettier -c  \"src/**/*.{js,jsx,css,html,scss}\"`

Reply:
Alright I'd add this also